### DISCUSSION TITLE
Переделать выбор устройств для передачи в Я в пользу нативного свича Expose

### DISCUSSION THREAD

**Discussion by pastukhov** [dext0r/yandex_smart_home#524] [Ideas]
Привет!
При большом списке устройств для передачи получается длинный скрол с галочками. Пользоваться этим не удобно.
В то же время в HA появился инструмент https://www.home-assistant.io/voice_control/voice_remote_expose_devices/
Как думаете, стоит на это переехать?

**Comment by dext0r** [comment]
https://github.com/dext0r/yandex_smart_home/issues/449

**Comment by pastukhov** [comment]
Прежде, чем запатчить список диалоговых асистенов [тут](https://github.com/home-assistant/core/blob/master/homeassistant/components/homeassistant/exposed_entities.py#L25), видимо, сначала придется стать полноценным диалоговым асистеном. Для этого предлагаю объединить усилия с https://github.com/gritaro/gigachain
Там уже реализовано общение с YandexGPT, но не сделано управление объектами.

**Comment by dext0r** [comment]
#559
